home-application/src/test/java/com/softserveinc/ita/homeproject/application/provider/JerseyParameterNameProviderTest.java

```diff
@@ -0,0 +1,67 @@
+package com.softserveinc.ita.homeproject.application.provider;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.withSettings;
+
+import java.lang.annotation.Annotation;
+import javax.ws.rs.CookieParam;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.MatrixParam;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+class JerseyParameterNameProviderTest {
```

Comment:
What do we test here?

Reply:
SonarCloud code analysis did not pass. This test was written to cover an unused functionality. If you delete the unused functionality (HeaderParam, CookieParam, FormParam, Context, MatrixParam) in the file (home-application / src / main / java / com / softserveinc / ita / homeproject / application / provider / JerseyParameterNameProvider.java), then the test will not be needed